test/e2e/autoscale.go

```diff
@@ -390,7 +404,5 @@ func AssertAutoscaleUpToNumPods(ctx *TestContext, curPods, targetPods float64, d
 		return checkPodScale(ctx, targetPods, minPods, maxPods, done, quick)
 	})
 
-	if err := grp.Wait(); err != nil {
-		ctx.t.Fatal("Error: ", err)
-	}
+	return grp
```

Comment:
I think it's somewhat unusual to pass errgroups around. Should we return `func() error` instead and just return the `Wait` method of the group? Seems to have a similar effect.

Reply:
OK. Why not.